### PR TITLE
Fix fname param to firstName in email templates

### PR DESCRIPTION
**Pull request by lilachritter**

### CODE REVIEW

**Comment by lilachritter** [comment]
Hey,
Encountered small 'typo' bug - emails where been sent without the name in the email top row.

**Comment by dortheimer** [comment]
@lilachritter , since Eyal decided that first name isn't required for registration, it doesn't exist any more. The new version I've pushed has a new email, in Hebrew and without the first name.

File: backend/api/service/email/newSignUp.mustache [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 <head>
   <title>Welcome to Meirim</title>
-  <link href="https://fonts.googleapis.com/css?family=Heebo" rel="stylesheet"> </link>
+<link href="https://fonts.googleapis.com/css?family=Heebo" rel="stylesheet"> </link>
```
Comment (BlueHotDog) [inline backend/api/service/email/newSignUp.mustache]: i think the prev indentation was correct.. if it's inside the head

File: backend/api/service/email/newAlert.mustache [hunk 2]
```diff
@@ -4,7 +4,7 @@
 </head>
 <body>
 
-Hi {{fname}}!
+Hi {{firstName}}!
```
Comment (BlueHotDog) [inline backend/api/service/email/newAlert.mustache]: yes for full names!

**Pull request closed without merging**